assets/prefabs/Raspberry/Raspberry.prefab

```diff
@@ -0,0 +1,10 @@
+{
+  "parent": "engine:iconItem",
+  "DisplayName": {
+    "name": "Raspberry"
+  },
+  "Item": {
+    "icon": "SimpleFarming:SimpleFarming#Raspberry",
```

Comment:
You need to use "SimpleFarming:Raspberry" to refer to a standalone image. "SimpleFarming:SimpleFarming#Raspberry" means you have a "SimpleFarming.png" image atlas with a single entry inside it named "Raspberry"